app/views/menus/_menu_item.html.haml

```diff
@@ -0,0 +1,7 @@
+%tr
+  %td
+    = f.input_field :_destroy, as: :hidden
+    = link_to t("menu_items.delete"), '#', class: 'remove_record'
+  %td= f.input :name, label: false, input_html: { size: 25 }
+  %td= f.input :price, label: false
+  %td= f.input :menu_type, collection: MenuItem::TYPES_OF_DISHES.map{|menu_item| [menu_item.to_s.humanize, menu_item]}, label: false, include_blank: false
```

Comment:
`MenuItem::TYPES_OF_DISHES.map{|menu_item| [menu_item.to_s.humanize, menu_item]}` this is waaay too much for the view. this should be in helper (at least) or in the facade (at best)